configs/train_config.json

```diff
@@ -12,14 +12,142 @@
     "here_we_need_some_variabes": true
   },
   "conveyor": {
```

Comment:
Let's put topology description outside of conveyor description. There is event a field called `topology` below for this purpose.